app/tests/semaphore_test.cc

```diff
@@ -76,6 +88,43 @@ TEST(SemaphoreTest, TimedWait) {
       0.20 * firebase::internal::kMillisecondsPerSecond);
 }
 
+#if SEMAPHORE_LINUX
```

Comment:
WDYT about just using #ifdef FIREBASE_ANDROID || FIREBASE_LINUX here and above? To me that's clearer, even though it's not the same as the check in semaphore.h.

Reply:
Done.